### DISCUSSION TITLE
dBeaver doesn't start after upgrade to 24.0.4 (zip download) - incompatible JNA native library

### DISCUSSION THREAD

**Discussion by pistata112** [dbeaver/dbeaver#33274] [Q&A] [answered]
### Description

dbeaver crash just after it Window shows with modal 
---------------------------
Dbeaver
---------------------------
An error has occurred. See the log file
D:\usr\OneDrive\work\O2\DBeaverData\.metadata\.log.
---------------------------
OK   
---------------------------

There is a a terror in .log., see whole log in attachment:
There is an incompatible JNA native library installed on this system
Expected: 7.0.0
Found:    6.1.6

### DBeaver Version

24.0.4

### Operating System

Windows 11

### Additional context

[.log](https://github.com/dbeaver/dbeaver/files/15251802/default.log)

**Comment by ShadelessFox** [comment, marked as answer]
Hello @pistata112,

Did you unpack the contents of the downloaded zip archive on top of the existing installation? You can't do that. Please perform a clean installation.

**Comment by chenweihua** [comment]
windows10同样的问题

**Comment by Ejfiou** [comment]
> Hello @pistata112,
> 
> Did you unpack the contents of the downloaded zip archive on top of the existing installation? You can't do that. Please perform a clean installation.

it works for me.

**Comment by pistata112** [comment]
Thanks for advice,
it works when zip content is copied to new directory (not rewriting dbeaver directory with previous version)
Thanks
maki

**Comment by bay007** [comment]
I have downloaded and performed a clean installation, it works great.
